src/material-experimental/mdc-chips/_mdc-chips.scss

```diff
@@ -2,6 +2,11 @@
 @import '../mdc-helpers/mdc-helpers';
 @import '@material/theme/functions';
 
+// Selector for the element that has a background color and opacity applied to its ::before and
+// ::after for state interactions (hover, active, focus). Their API calls this their
+// "ripple target", but we do not use it as our ripple, just state color.
+$mat-chip-state-target: '.mdc-chip__ripple';
```

Comment:
It doesn't look like this variable is being used anywhere. If it's just here for the comment it could end up being removed if somebody notices that it isn't being used anymore. Maybe we should put the comment either in one of the other files or next to the element inside the HTML?

Reply:
Right, good catch - this didn't end up getting used as I expected. Removed